FreeRTOS/iot-reference-esp32c3
# Move OTA max data buffer number to project configurations.

Move OTA max data buffer number to project configurations.

Description
-----------

As #96, #97, #98 describe, there are compile error messages with OTA demo enabled.
- <ROOT>/components/FreeRTOS-Libraries-Integration-Tests/FreeRTOS-Libraries-Integration-Tests/src/ota_pal_test.cmake not found, and that file is removed at the latest release.
- Missing `otaconfigMAX_NUM_OTA_DATA_BUFFERS` definition.

Test Steps
-----------

1. idf.py menuconfig (Featured FreeRTOS IoT Integration -> Enable OTA demo -> check)
1. idf.py build

Checklist:
----------


- [NA] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

#96, #97, #98

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
